src/lib/utils/recaptcha.ts

```diff
@@ -0,0 +1,13 @@
+const secretKey = RECAPTCHA_SECRET_KEY
+import { RECAPTCHA_SECRET_KEY } from '$env/static/private'
+
+export async function recaptchaResponse(schema: string) {
```

Comment:
You shouldn't pass the schema, that's part of the action validation process, here you should just receive the token.